magento/magento2
# Remove wrong test expectation

### Description (*)
Removing wrong test expectation which fails unit tests

### Fixed Issues (if relevant)
1. Fixes https://github.com/magento/magento2/issues/12419

### Manual testing scenarios (*)
1. Run `bin/magento dev:tests:run unit`, which fails with:

```
There was 1 failure:

1) Magento\ProductVideo\Test\Unit\Controller\Adminhtml\Product\Gallery\RetrieveImageTest::testExecute
Expectation failed for method name is equal to <string:getDirectoryWrite> when invoked 1 time(s).
Method was expected to be called 1 times, actually called 0 times.
```

### Contribution checklist (*)
 - [ x ] Pull request has a meaningful description of its purpose
 - [ x ] All commits are accompanied by meaningful commit messages
 - [ x ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ x ] All automated tests passed successfully (all builds on Travis CI are green)
